.github/workflows/awsfulltest.yml

```diff
@@ -5,6 +5,11 @@ name: nf-core AWS full size tests
 on:
   release:
     types: [published]
+  # TODO: just for testing, remove again
+  push:
+    branches:
+      - dev
+  pull_request:
```

Comment:
```suggestion
```

Reply:
Hi Sabrina, the AWS tests will not run on pull requests as they require the organization secrets. So we can remove this here :smile: